routes/frontend.php

```diff
@@ -1,3 +1,21 @@
 <?php
 
-// Here you can add routes for the frontend.
+use Astrogoat\Blog\Http\Controllers\ArticleController;
+use Astrogoat\Blog\Http\Controllers\CategoryController;
+use Illuminate\Support\Facades\Route;
+
+Route::group([
+    'as' => 'blog.article.',
+    'prefix' => 'blog/article/',
+//    'middleware' => ['enabled:Astrogoat\Blog\Settings\BlogSettings']
+], function () {
+    Route::get('{article:slug}', [ArticleController::class, 'show'])->name('show');
+});
+
+Route::group([
+    'as' => 'blog.category.',
+    'prefix' => 'blog/category/',
+//    'middleware' => ['enabled:Astrogoat\Blog\Settings\BlogSettings']
```

Comment:
Same as for the backend routes files. 
You can nest the groups, and the "as" should be pluralized.